src/main/java/racingcar/view/ConsoleOutput.java

```diff
@@ -0,0 +1,46 @@
+package racingcar.view;
+
+import racingcar.model.Car;
+
+import java.util.List;
+
+public class ConsoleOutput {
+
+    private static final String RACING_RESULT_MESSAGE = "레이싱 게임 결과입니다.";
+    private static final String RAIL = "-";
+    private static final String STATUS_FORMAT = "%s : %s%n";
+    private static final String COMMA = ", ";
+    private static final String WIN_MESSAGE = "가 최종 우승했습니다.";
+
+    public void resultMessage() {
+        System.out.println(RACING_RESULT_MESSAGE);
+    }
+
+    public void status(List<Car> cars) {
+        cars.forEach(
+                carStatus -> System.out.printf(STATUS_FORMAT, carStatus.getName(), rails(carStatus.getPosition())));
+        newLine();
+    }
+
+    private String rails(int position) {
+        return RAIL.repeat(Math.max(0, position));
+    }
+
+    public void winner(List<Car> cars) {
+        for (int i = 0; i < cars.size(); i++) {
+            printComma(i);
+            System.out.print(cars.get(i).getName());
+        }
```

Comment:
```suggestion
        System.out.println(
            cars.stream()
                .map(it -> it.getName())
                .collect(Collectors.joining(", "))
        );
```

stream을 적극 활용해 보면 좋을 것 같아요.